chaco/scatterplot.py

```diff
@@ -526,7 +526,7 @@ def _outline_color_changed(self):
         self.invalidate_draw()
         self.request_redraw()
 
-    def _either_metadata_changed(self):
+    def _either_metadata_updated(self, event):
```

Comment:
just to confirm, this method was inherited?

Reply:
Yep, from BaseXYPlot